source/tools/testConsistency/ClientApp/app.test.js

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// Licensed under the MIT License. 
+// To run these tests, first ensure that the flask app for nodeJs is running at 127.0.0.1 port 5000
+// and the flask app for .NET WPF is running at 127.0.0.1 port 4000. 
+const request = require("supertest");
+const app = require("./app");
+const fs = require("fs");
+
+test("The GET request to the index page should return a response code 200", function(done) {
+     request(app)
+      .get("/")
+      .then(function(response) {
+        expect(response.statusCode).toBe(200);
+        done();
+      });
+});
+
+test("Sending no card data should still return a response code 200", function(done) {
+    const templateBuffer = fs.readFileSync("../../../../samples/v1.0/Scenarios/ActivityUpdate.json");
+    const template = templateBuffer.toString('utf8')
+    request(app)
+      .post("/")
+      .send(cardTemplate=template, cardData="")
+      .then(function(response){
+          expect(response.statusCode).toBe(200);
+          done();
+      });
+});
+
+test("Sending no card template should still return a response code 200", function(done) {
+    const data = {
+                    "name": "Matt Hidinger"
```

Comment:
>Matt Hidinger [](start = 29, length = 13)

:) I think we can cook up a different name here instead of using Matt's yet again (also privacy in general though not that big of a deal since he's everywhere :))